### DISCUSSION TITLE
Runtime for cognito user pool function

### DISCUSSION THREAD

**Discussion by nick-zh** [serverless/serverless#11300] [Q&A] [answered]
is there a way i can influence the runtime for my `custom-resource-existing-cup` function?
E.g. i have set node16 for my functions, but this one is create as node14.
I can change it manually, just wanted to know if there is another way :smile:

**Comment by pgrzesik** [comment, marked as answer]
Hey @nick-zh - there's no "native" way to do so, it's currently hardcoded to use Node 14 runtime. I believe the only way to do so would be to use override https://www.serverless.com/framework/docs/providers/aws/guide/resources#override-aws-cloudformation-resource with logical ID of Lambda function used by that custom resource.
